### PR TITLE
[ASCN-382] Split secrets for Exceptional and Status

### PR DESCRIPTION
**Pull request by WouterDeKort**
This PR is for [ASCN-382](https://stackoverflow.atlassian.net/browse/ASCN-382).

DBRE uses two sets of credentials for OpServer:
- One for Exceptions (e.g. https://opserver.ascn-dev.int.gcp.stackoverflow.net/exceptions)
- One for SQL Status (e.g. https://opserver.ascn-dev.int.gcp.stackoverflow.net/sql/instance?node=db.db)

This PR adds a second set of secrets and splits the Sql secrets we use for Exceptional and Sql Status so we can configure this correctly in prod. For `ascn-dev`, I added the required secrets and deployed successfully from local CNAB.

I also asked Aaron if each app should have its own Exceptional credentials [here](https://stackexchange.slack.com/archives/C05PV8H4M6U/p1731580798978979) since I think we now share the `db-Exceptions-User` between all apps and I don't think that's correct.

[ASCN-382]: https://stackoverflow.atlassian.net/browse/ASCN-382?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (stephen-vakil) [inline .vscode/launch.json:1]: was this intentionally added?

**Review by stephen-vakil — APPROVED** [review]
verified locally with docker;  looks correct and works

File: .vscode/launch.json (hunk 1 above)
Comment (WouterDeKort) [reply .vscode/launch.json:1]: Yes. Is that ok?

Comment (stephen-vakil) [reply .vscode/launch.json:1]: i don't mind, just wanted to make sure it was intentional :)

File: cnab/app/build-app-image.ps1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 function Build-Local-App-Image() {
 
       
-    docker build -t local.software/stackeng/opserver/opserver -t cr.stackoverflow.software/stackeng/opserver/opserver:local .
+    # docker build -t local.software/stackeng/opserver/opserver -t cr.stackoverflow.software/stackeng/opserver/opserver:local .
```
Comment (aalear) [inline cnab/app/build-app-image.ps1:4]: Is this intentionally commented out?

**Review by aalear — APPROVED** [review]

**Pull request merged**